contributing.md

```diff
@@ -512,6 +512,97 @@ coverage report. You'll need to have [lcov](http://ltp.sourceforge.net/coverage/
 
         firefox coverage/index.html
 
+### Sanitizers
+
+Sanitizers capture very detailed information during runtime about code quality issues and print them to stderr.
+
+#### Install dependencies
+
+Install and enable Colcon mixin package:
+
+```bash
+sudo apt-get install python3-colcon-mixin
+colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+colcon mixin update default
+```
+
+Install colcon-sanitizer-reports (Recommended):
+
+```bash
+git clone https://github.com/colcon/colcon-sanitizer-reports.git
+cd colcon-sanitizer-reports
+sudo python3 setup.py install
+```
+
+You can find more details [here](https://github.com/colcon/colcon-sanitizer-reports/blob/master/README.rst) about the instalation process.
+
+#### Compiling the code and running the tests
+
+Create a Ignition workspace for ASAN/TSAN related tasks
+
+```bash
+mkdir -p ~/ignition/src
+cd ~/ignition
+wget https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-<Ignition distro>.yaml
+vcs-import src < collection-<Ignition distro>.yaml
+```
```

Comment:
How about linking to the source install document instead of repeating this block here? Otherwise we'll need to explain here how to install wget, vcs, etc. It's easier to keep that in one place.